### ISSUE TITLE
autosummary unexpectedly strips docstrings containing "i.e."

### ISSUE THREAD

**Issue by lsaffre** [sphinx-doc/sphinx#4689]
When I have a docstring "blabla, i.e. blablabla", autosummary trims it to "blabla, i." 
Until 1.7.0 (before #4563 was fixed) autosummary truncated to "blabla, i.e.". Both truncations are a problem.

To reproduce the problem, write the following function in a module which is documented using autosummary:
```
def myfunc(arg):
    """Blabla, i.e. bla.
    """
    pass
```

**Comment by astrofrog** [comment]
I am seeing a related issue in a number of packages - for example I have a method with the following as the first line of the docstring:

```
    References a :class:`~glue.core.component.Component` object within a :class:`~glue.core.data.Data` object.
```

causes warnings such as:

```
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
```

Is this the same underlying problem or shall I open a separate issue?

**Comment by lsaffre** [comment]
I see these warnings as well with 1.7.1 and they equally seem to be
caused by the the autosummary table, but I'd say that they are worth a
separate issue.

**Comment by tk0miya** [comment]
@astrofrog The problem you mentioned is already reported as #4685. And I'm working for it. Please wait a moment.

**Comment by tk0miya** [comment]
Fixed by #4694.
Thank you for reporting
